### PR TITLE
integration tests with restify and restify-clients

### PR DESCRIPTION
**Pull request by DonutEspresso**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -506,4 +506,65 @@ describe('restify-errors node module.', function() {
...
+
+            client.get('/1', function(err, req, res, data) {
+                assert.ok(err);
+                assert.equal(res.statusCode, 404);
+                assert.equal(data.message, 'gone girl');
+                done();
+            });
+        });
+
+        it('should send send RestErrors with status codes', function(done) {
```
Comment (micahr) [inline test/index.js]: how about with `status status` codes :)

**Comment by micahr** [comment]
:+1:

File: test/index.js (hunk 1 above)
Comment (DonutEspresso) [inline test/index.js]: HA! eyes of a hawk.

Comment (micahr) [inline test/index.js]: hawkeyes

Comment (DonutEspresso) [inline test/index.js]: O_O

**Pull request merged**